src/container/srv_target.c

```diff
@@ -410,8 +433,8 @@ cont_child_alloc_ref(void *co_uuid, unsigned int ksize, void *po_uuid,
 		goto out_pool;
 
 	uuid_copy(cont->sc_uuid, co_uuid);
-	cont->sc_aggregation_min = DAOS_EPOCH_MAX;
 	/* prevent aggregation till snapshot iv refreshed */
+	cont->sc_aggregation_start_hlc = 0;
```

Comment:
please move the comment down before 'sc_aggregation_max' or move this change up above the comment, the comment 'prevent ....' is for sc_aggregation_max.